recipes/xpdtools/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "xpdtools" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "d730756e96368bd050a16d07a4e1f0563789aabbb753dec2dba7be29728ae3b3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://github.com/xpdAcq/{{ name }}/archive/v{{ version }}.tar.gz
```

Comment:
Looks like the link is 404'ing and the correct link is `/{{ version }}.tar.gz` instead of `/v{{ version }}.tar.gz`.